### PR TITLE
Prettier

### PR DESCRIPTION
**Pull request by gonziesc**
## Summary
- Adding prettier to eslint
- Fixed spaces and indentation

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -5,52 +5,41 @@
     "mocha": true,                   // mocha keywords
     "es6": true
   },
+  "plugins": [
+    "prettier"
+  ],
+  "extends": [
+    "prettier"
+  ],
   "rules": {
```
Comment (sbalay) [inline .eslintrc]: I'd delete all these rules and use prettier config for code formatting rules and airbnb standard for code quality rules. @mishuagopian thoughts?

Comment (sbalay) [inline .eslintrc]: I'll explain myself. Almost every linter, including eslint, have two kind of rules:
- Formatting rules: These restrict the way we write code, for example: indent, arrow-spacing, max-len, etc.
- Code quality rules: These rules restrict the way we organize and execute code, for example: complexity, default-case, constructor-super, etc.

Formatting rules enforce code style consistency while code quality rules enforce good practices and avoids anti-patterns.

In my previous comment I suggest leaving all the setup of formatting rules to prettier, which is a code formatter that has an opinionated and widely approved standard. We still need to take care of code quality rules, my suggestion here is using another standard that's widely approved by the community, which is airbnb's javascript guide.

Comment (sbalay) [inline .eslintrc]: **TL;DR**
```js
// .eslintrc.js
module.exports = {
  env: {
    node: true,
    mocha: true,
    es6: true
  },
  parser: "babel-eslint",
  parserOptions: {
    ecmaFeatures: {
      experimentalObjectRestSpread: true
    },
    sourceType: "module"
  },
  plugins: ["prettier"],
  extends: [
    "eslint:recommended",
    "airbnb-base",
    "prettier"
  ],
  globals: {
    __DEV__: true
  },
  rules: {
    "prettier/prettier": ["error"],
  }
};
```

Comment (sbalay) [inline .eslintrc]: I'd also override some of prettier defaults:
```
"prettier/prettier": ["error", { printWidth: 110, singleQuote: true }]
```

Comment (mishuagopian) [inline .eslintrc]: Agree :+1:

Comment (gonziesc) [inline .eslintrc]: Great!

Comment (gonziesc) [inline .eslintrc]: @mishuagopian what do you think about the rules you have set to be ignored like http://eslint.org/docs/rules/one-var?

**Comment by sbalay** [comment]
@gonziesc ready! sorry for the delay!

**Comment by sbalay** [comment]
@mishuagopian ci failure does not make a lot of sense. could you check?

**Pull request merged**